Assets/MoveBase.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class MoveBase : MonoBehaviour
+{
+    public enum EMoveType
+    {
+        None = 0,
+        Left,
+        Right,
+        Up,
+        Down,
+    }
+
+    public enum EBlockLayerType
+    {
+        Player = 7,
+        Enemy = 8,
+        Ground = 6,
+    }
```

Comment:
```suggestion
    public enum ECharacterLayerType
    {
        Ground = 6,
        Player = 7,
        Enemy = 8,
    }
```
微々たることかもしれませんが、
数字の順番が並んでいないと変に目立って見えてしまいます。
恐れ入りますが順番通りに並べてください。
また、名前として「Block」とついていますが、
キャラクターのレイヤーですので、
「Character」の方がよろしいかと思います。
ご検討をお願いします。m(´・ω・｀)m

Reply:
こちら、将来のレイヤー増が見込まれるため、順番通りにならないよう、あえてずらしていますので、このまま通していただければとおもいますー　ありがとうございます！